zipkinCore.thrift

```diff
@@ -0,0 +1,81 @@
+# Copyright 2012 Twitter Inc.
```

Comment:
ps we just recently moved the authoritative copy here: https://github.com/openzipkin/zipkin-api/tree/master/thrift